### PR TITLE
Feature namespace refactor obsolete class removal

### PR DESCRIPTION
**Pull request by DDReaper**
Overview
---
Removal of all obsolete classes / functions /properties

Changes
---
Removed all code marked as obsolete with one exception

- uAudioManager has an Obsolete property, which when removed causes issues.  This should be resolved.

### CODE REVIEW

File: Assets/MixedRealityToolkit-UnitTests/Editor/Utilities/Extensions/GameObjectExtensionsTests.cs [hunk 1]
```diff
@@ -221,7 +221,7 @@ public void GetFullPathTests(string result, params string[] names)
             }
 
 #pragma warning disable 618
-            Assert.That(parent.GetFullPath(), Is.EqualTo(result));
+            Assert.That(parent.transform.GetFullPath(), Is.EqualTo(result));
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-UnitTests/Editor/Utilities/Extensions/GameObjectExtensionsTests.cs]: What obsolete code is the 618 suppressing?

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-UnitTests/Editor/Utilities/Extensions/GameObjectExtensionsTests.cs]: We can most likely remove the 618

Comment (DDReaper) [reply Assets/MixedRealityToolkit-UnitTests/Editor/Utilities/Extensions/GameObjectExtensionsTests.cs]: Removed

**Comment by DDReaper** [comment]
Raised issue #1681 for uAudioManager issue, to be resolved later.
Merging

**Comment by StephenHodgson** [comment]
Yeah the uAudioManager one has some logic that updates it to use the audio bank features. 

That one is a bit tricky.

**Pull request merged**